rust/operator-binary/src/framework/role_group_utils.rs

```diff
@@ -65,6 +65,16 @@ impl ResourceNames {
 
         format!("{}{SUFFIX}", self.qualified_role_group_name())
     }
+
+    pub fn listener_service_name(&self) -> String {
```

Comment:
This code is okay for now, but I will probably change it in the future.

The problem is that the caller must know how the returned string may be used or not. For instance, `listener_service_name` can be used as an annotation value but not as a label value (which is limited to 63 characters).